subsys/bluetooth/services/ots/ots_client.c

```diff
@@ -1095,6 +1120,27 @@ static void write_oacp_cp_cb(struct bt_conn *conn, uint8_t err,
 
 	inst->busy = false;
 }
+static void write_oacp_cp_write_req_cb(struct bt_conn *conn, uint8_t err,
```

Comment:
newline, alignment

Reply:
@pirun still a few unaligned lines, check the diff in github to see what I'm talking about

Reply:
Please check again.